litmus-portal/frontend/src/pages/MonitoringDashboardPage/styles.ts

```diff
@@ -111,6 +121,211 @@ const useStyles = makeStyles((theme) => ({
   modal: {
     padding: theme.spacing(15, 0),
   },
+
+  selectDate: {
+    display: 'flex',
+    height: '2.9rem',
+    minWidth: '9rem',
+    marginLeft: theme.spacing(3.75),
+    border: `0.1px solid ${theme.palette.border.main}`,
+    background: theme.palette.background.paper,
+    borderRadius: 4,
+    textTransform: 'none',
+  },
+
+  displayDate: {
+    width: '100%',
+    color: theme.palette.text.primary,
+  },
+
+  rangeSelectorIcon: {
+    width: '0.625rem',
+    height: '0.625rem',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(-1),
+  },
+
+  rangeSelectorClockIcon: {
+    width: '0.825rem',
+    height: '0.825rem',
+    marginLeft: theme.spacing(-1),
+    marginRight: theme.spacing(1),
+  },
+
+  rangeSelectorPopover: {
+    marginTop: theme.spacing(37.5),
```

Comment:
margin spacing of 37.5 is very high. Please rearrange the parent div to avoid this.

Reply:
Its not possible due to the local component, i am developing in litmus-ui to replace that, current one has some bug with respect to orientation.

Reply:
we can use percentage here, thats lesser around 10 15..

Reply:
If you are working on local component then first fix all the bugs locally first. Otherwise remove the feature for the PR

Reply:
Its not possible to remove it as this is not adding the feature, it merely adding the logic and an existing component to test the changes, as discussed the litmus-ui component will be replacing this temporary component.

Reply:
cc: @arkahjyotiMukherjee

Reply:
Litmus-ui will have the same component right...why not build it locally in a good way and take it to litmus-ui...like we did with the Litmus-Stepper component